### PR TITLE
[coop] Enable by default

### PR DESCRIPTION
**Pull request by luhenry**
Part of #6921

### CODE REVIEW

**Comment by vargaz** [comment]
We should investigate the perf implications of this and fix any major problems before enabling it.

**Comment by luhenry** [comment]
@monojenkins build pkg

**Comment by lewurm** [comment]
that's probably the first time in history someone runs the interpreter with  coop 😛 apparently it needs some fixes, but not too bad! safepoints are not supported in the interpreter at the moment

**Comment by marek-safar** [comment]
/cc @migueldeicaza are you ok with the default change

@vargaz I agree and I asked @luhenry for some real scenarios numbers

File: man/mono.1 [hunk 1]
```diff
@@ -1160,18 +1160,18 @@ For platforms that do not otherwise have a way of obtaining random bytes
 this can be set to the name of a file system socket on which an egd or
 prngd daemon is listening.
 .TP
-\fBMONO_ENABLE_COOP\fR
-This makes the Mono runtime and the SGen garbage collector run in cooperative
-mode as opposed to run on preemptive mode.   Preemptive mode is the mode
+\fBMONO_DISABLE_COOP\fR
```
Comment (migueldeicaza) [inline man/mono.1]: My feeling is that we should preserve both environment variables, in case things change in the future

**Comment by migueldeicaza** [comment]
I looked at the original issue, if the intention is to start measuring, changing the default looks premature.

What is the plan to measure without making the change?

**Comment by luhenry** [comment]
This is going to be done through Hybrid Suspend

**Pull request closed without merging**